scalatest/src/main/scala/org/scalatest/exceptions/StackDepthExceptionHelper.scala

```diff
@@ -140,7 +140,7 @@ private[scalatest] object StackDepthExceptionHelper extends Serializable {
   }
 
   def getStackDepthFun(fileName: String, methodName: String, adjustment: Int = 0): (StackDepthException => Int) = { sde =>
-    getStackDepth(sde.getStackTrace, fileName, methodName, adjustment)
+    getStackDepth(sde.getStackTrace.map(_.nn), fileName, methodName, adjustment)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.